### ISSUE TITLE
Issue with "make install"

### ISSUE THREAD

**Issue by laurvana** [sabrishamcar/Data550_finalproject#1]
Hi Sarina and  @benkeser, 

I think this is more of a me issue, but I am attaching a screenshot of the error I get when I run the `make install` command. The error is much longer than I can include in one screenshot but I attached the ending message.

Thanks,
Lauren

<img width="857" alt="Screenshot 2024-04-16 at 4 46 58 PM" src="https://github.com/sabrishamcar/Data550_finalproject/assets/162159288/3b6863f2-0d3e-45cf-9385-438efc61d5d8">

**Comment by benkeser** [comment]
yeah seems like a compiler problem on your end. No worries. Thanks for confirming.
